src/main/java/roomescape/reservation/domain/ReservationPeriod.java

```diff
@@ -0,0 +1,34 @@
+package roomescape.reservation.domain;
+
+import java.time.LocalDate;
+
+public class ReservationPeriod {
+
+    private final LocalDate standardDate;
+    private final int startOffset;
+    private final int endOffset;
+
+    public ReservationPeriod(LocalDate standardDate, int startOffset, int endOffset) {
+        validateOffset(startOffset,endOffset);
+        this.standardDate = standardDate;
+        this.startOffset = startOffset;
+        this.endOffset = endOffset;
+    }
+
+    public LocalDate findStartDate() {
+        return standardDate.minusDays(startOffset);
+    }
+
+    public LocalDate findEndDate() {
+        return standardDate.minusDays(endOffset);
+    }
+
+    private void validateOffset(int startOffset, int endOffset){
```

Comment:
```suggestion
        validateOffset(startOffset, endOffset);
        this.standardDate = standardDate;
        this.startOffset = startOffset;
        this.endOffset = endOffset;
    }

    public LocalDate findStartDate() {
        return standardDate.minusDays(startOffset);
    }

    public LocalDate findEndDate() {
        return standardDate.minusDays(endOffset);
    }

    private void validateOffset(int startOffset, int endOffset) {
```
코드 컨벤션에 맞지 않는 것은 없는지 항상 확인하는 습관이 필요할 것 같아요.